api.py

```diff
@@ -223,6 +223,39 @@ def _get_all_pods():
         pods.append(i.metadata.name)
     return pods
 
+# work in progress for log stream endpoint
+@api.route("/stream_logs")
+def stream_logs_html(req, resp):
+    """Route to stream logs"""
+    resp.content = api.template(
+        'stream_logs.html',
+        log_stream=_get_log_stream()
+    )
+
+def _get_log_stream():
+
+# Not sure where this block should go, and what items we actually need
+    # Do I need this line or is equivalent to 'k8s_client.read_namespaced_pod_log'?
+    api_instance = kubernetes.client.CoreV1Api(kubernetes.client.ApiClient(configuration))
```

Comment:
You don't need this. There's already a kubernetes client instantiated earlier. Use that.